From e5064de3d89f00bb66e73883aacda898cbad0d2a Mon Sep 17 00:00:00 2001
From: Marc Hanheide <marc@hanheide.net>
Date: Wed, 15 Oct 2014 09:10:28 +0100
Subject: [PATCH 1/3] added sourcing of mira setup file

---
 debian/rules | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/debian/rules b/debian/rules
index 5d802a0..2c2b995 100755
--- a/debian/rules
+++ b/debian/rules
@@ -24,6 +24,7 @@ override_dh_auto_configure:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
+	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_configure -- \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="/opt/ros/hydro" \
@@ -34,6 +35,7 @@ override_dh_auto_build:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
+	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_build
 
 override_dh_auto_test:
@@ -42,6 +44,7 @@ override_dh_auto_test:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	echo -- Running tests. Even if one of them fails the build is not canceled.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
+	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_test || true
 
 override_dh_shlibdeps:
@@ -49,6 +52,7 @@ override_dh_shlibdeps:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
+	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_shlibdeps -l$(CURDIR)/debian/ros-hydro-scitos-mira//opt/ros/hydro/lib/
 
 override_dh_auto_install:
@@ -56,4 +60,5 @@ override_dh_auto_install:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
+	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_install
-- 
2.0.4

