From 8d4d86081093f762f8b16990184ecd60dc97ffe0 Mon Sep 17 00:00:00 2001
From: Marc Hanheide <marc@hanheide.net>
Date: Wed, 15 Oct 2014 10:06:22 +0100
Subject: [PATCH 2/3] set env variables for MIRA

---
 debian/rules | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/debian/rules b/debian/rules
index 2c2b995..1e2e99b 100755
--- a/debian/rules
+++ b/debian/rules
@@ -16,6 +16,10 @@ export DH_OPTIONS=-v --buildsystem=cmake
 export LDFLAGS=
 export PKG_CONFIG_PATH=/opt/ros/hydro/lib/pkgconfig
 
+export MIRA_PATH=/opt/mira-scitos
+export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH):/opt/mira-scitos/lib
+export PATH=$(PATH):/opt/mira-scitos/bin
+
 %:
 	dh  $@
 
@@ -24,7 +28,6 @@ override_dh_auto_configure:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
-	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_configure -- \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="/opt/ros/hydro" \
@@ -35,7 +38,6 @@ override_dh_auto_build:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
-	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_build
 
 override_dh_auto_test:
@@ -44,7 +46,6 @@ override_dh_auto_test:
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	echo -- Running tests. Even if one of them fails the build is not canceled.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
-	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_test || true
 
 override_dh_shlibdeps:
@@ -52,7 +53,6 @@ override_dh_shlibdeps:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
-	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_shlibdeps -l$(CURDIR)/debian/ros-hydro-scitos-mira//opt/ros/hydro/lib/
 
 override_dh_auto_install:
@@ -60,5 +60,4 @@ override_dh_auto_install:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/hydro/setup.sh" ]; then . "/opt/ros/hydro/setup.sh"; fi && \
-	if [ -f "/opt/mira-scitos/.bashrc" ]; then . "/opt/mira-scitos/.bashrc"; fi && \
 	dh_auto_install
-- 
1.9.1

